submissions/dashakim/tiny-world/index.js

```diff
@@ -8,40 +8,32 @@
 // Define your objects here
 class Inhabitant {
     constructor(species, name, gender, legs, hands, saying) {
-        this.species = species;
-        this.name = name;
-        this.gender = gender;
-        this.legs = legs;
-        this.hands = hands;
-        this.saying = saying
+      this.species = species;
+      this.name = name;
+      this.gender = gender;
+      this.legs = legs;
+      this.hands = hands;
+      this.saying = saying;
     }
-}
-
-const man = new Inhabitant('human', 'Leonid', 'male', '2', '2', 'Hello everyone!')
-const woman = new Inhabitant('human', 'Daria', 'female', '2', '2', 'Good day!')
-const dog = new Inhabitant('dog', 'Archi', 'male', '4', '0', 'Woof-woof')
-const cat = new Inhabitant('cat', 'Co-co', 'female', '4', '0', 'Meow')
-const catWoman = new Inhabitant('cat-woman', 'Kitty', 'female', '2', '2')
-man.friends = [woman.name, dog.name].join(', ')
-woman.friends = [cat.name]
-cat.friends = [man.name, woman.name].join(', ')
-dog.friends = [man.name, woman.name, cat.name].join(', ')
-catWoman.saying = cat.saying;
-catWoman.friends = cat.friends
-
-const showIn = function (obj) {
-    let result = ''
-    for (let key in obj) {
-        result += `${key}: ${obj[key]}; `
+  }
+  
+  const man = new Inhabitant("human", "Leonid", "male", "2", "2", "Hello!");
+  const woman = new Inhabitant("human", "Daria", "female", "2", "2", "Good day!");
+  const dog = new Inhabitant("dog", "Archi", "male", "4", "0", "Woof-woof");
+  const cat = new Inhabitant("cat", "Co-co", "female", "4", "0", "Meow");
+  const catWoman = new Inhabitant("cat-woman", "Kitty", "female", "2", "2");
+  catWoman.saying = cat.saying;
+  const inhabitant = [man, woman, cat, dog, catWoman];
+  
+  const showInfo = function(obj) {
+    let result = "";
+    for (let i in obj) {
+      result += `${i}: ${obj[i]}; `;
     }
-    return result
-}
-
-print(showIn(man))
-print(showIn(woman))
-print(showIn(cat))
-print(showIn(dog))
-print(showIn(catWoman))
+    return result;
```

Comment:
The following snippet is equivalent to this function body:
```
return Object.keys(obj).map(key => `${i}: ${obj[i]}`).join('; ');
```
Yet another approach:
```
return Object.entries(obj).map(entry => `${entry[0]}: ${entry[1]}`).join('; ');
```
which can be further refactored to be  even more semantic:
```
return Object.entries(obj).map(([key, value]) => `${key}: ${value}`).join('; ');
```

Consider investigating the snippets above and find out the power of [`Object`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object#Methods_of_the_Object_constructor) and [`Array instance`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array#Instance_methods) methods.

Those make code concise and semantically clear.